server/src/main/java/io/littlehorse/server/auth/OAuthServerAuthorizer.java

```diff
@@ -3,15 +3,16 @@
 import com.google.common.base.Strings;
 import com.google.common.cache.Cache;
 import com.google.common.cache.CacheBuilder;
-import io.grpc.Metadata;
-import io.grpc.ServerCall;
-import io.grpc.ServerCallHandler;
-import io.grpc.Status;
+import io.grpc.*;
```

Comment:
There is a setting on VSCode which allows you to avoid this.
```
    "java.sources.organizeImports.starThreshold": 200,
    "java.sources.organizeImports.staticStarThreshold": 200
```